src/Components/SalesforceWrapper.tsx

```diff
@@ -57,27 +57,29 @@ export const SalesforceWrapper: React.FC<SalesforceWrapperProps> = ({
     )}/embeddedservice/5.0/esw.min.js`,
     removeOnUnmount: true,
     onReady: () => {
-      window.embedded_svc.settings.defaultMinimizedText = "Chat"
-      window.embedded_svc.settings.enabledFeatures = ["LiveAgent"]
-      window.embedded_svc.settings.entryFeature = "LiveAgent"
+      if (window.embedded_svc?.settings?.enabledFeatures) {
```

Comment:
I think if we check against settings that should enable all of the rest of the props below to be guarded safely:

```
if (window.embedded_svc?.settings) {
  ...
}
```

It doesn't look like there's anything deeper than `settings.foo`, so all good 👍